tests/l2/README.md

```diff
@@ -73,8 +73,10 @@ This workload runs [qatlib](https://github.com/intel/qatlib) sample tests. Refer
 
 *	Deploy and execute the workload.
 
-    * Run with IPC_LOCK capability for pod
+    * Run with IPC_LOCK capability for pod using custom SCC and rbac with service account
   
+```$ oc apply -f https://raw.githubusercontent.com/intel/intel-technology-enabling-for-openshift/main/security/qatlib_scc.yaml```
+
 ```$ oc apply -f https://raw.githubusercontent.com/intel/intel-technology-enabling-for-openshift/main/security/qatlib_rbac.yaml```
```

Comment:
Create the intel-qat service account using intel-qat-scc

Reply:
changed it to Create the intel-qat service account to use intel-qat-scc